charts/prometheus-postgresql-alerts/values.yaml

```
@@ -0,0 +1,75 @@
+global:
+  additionalRuleLabels: []
+
+defaultRunbookUrl: https://qonto.github.io/database-monitoring-framework/tree/{{chartVersion}}/runbooks/postgresql/{{alertName}}
+rulesGroupName: postgresql.rules
+
+rules:
+
+  PostgreSQLExporterDown:
+    expr: up{} * on (instance) postgres_exporter_build_info{} < 1
+    for: 5m
+    labels:
+      severity: critical
+    annotations:
+      summary: "{{ $labels.instance }} exporter is down"
+
+  PostgreSQLExporterScraping:
```

```suggestion
  PostgreSQLExporterScrapingTooLong:
```

Regarding @dcupif 's comment, we should have an hint of the issue in the alert reference.